crates/sui-rpc-api/generate-grpc-docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+set -e
+
+SCRIPT_PATH=$(realpath "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
+
+PROTO_FILES=(
+proto/sui.node.v2.proto
+proto/sui.types.proto
+proto/google/protobuf/empty.proto
+proto/google/protobuf/timestamp.proto
+)
+
+# requires that protoc as well as the protoc-gen-doc plugin is installed and
+# available on $PATH. See https://github.com/pseudomuto/protoc-gen-doc for more
+# info
+cd "$SCRIPT_DIR" && protoc --doc_out=proto/ --doc_opt=markdown,documentation.md ${PROTO_FILES[@]} --proto_path=proto/
```

Comment:
does this need to get run manually after any proto change to re-gen the docs? if yes, should we maybe add a note to remind people to do this? (can be deferred to a point in time where more than just you are making changes to the proto files)

Reply:
yes right now it  needs to be run manually. I didn't want to create any CI enforcement yet since i'm still working with @ronny-mysten to figure out how we actually want to use this documentation.